cmc-ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ContactDetailsMapper.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import uk.gov.hmcts.cmc.ccd.domain.ContactDetails;
+
+public class ContactDetailsMapper
+    implements Mapper<ContactDetails, uk.gov.hmcts.cmc.domain.models.legalrep.ContactDetails> {
+
+    @Override
+    public ContactDetails to(uk.gov.hmcts.cmc.domain.models.legalrep.ContactDetails contactDetails) {
+        return ContactDetails.builder()
+            .email(contactDetails.getEmail().orElse(null))
+            .phone(contactDetails.getPhone().orElse(null))
+            .dxAddress(contactDetails.getDxAddress().orElse(null))
+            .build();
+    }
+
+    @Override
+    public uk.gov.hmcts.cmc.domain.models.legalrep.ContactDetails from(ContactDetails contactDetails) {
```

Comment:
Can we change the name to avoid fully qualified imports in code?

Reply:
I think it is ok, it is hidden away

Reply:
How is it hidden away? The class which will use this mapper will need to use a fully qualified name.

Reply:
@rlewan What suggestion do you have for naming these?

Reply:
Prefix with `CCD`.